testing/tests/DevExpress.ui.widgets.editors/validationMessage.tests.js

```diff
@@ -33,14 +33,19 @@ QUnit.module('options', moduleSetup, () => {
     });
 
     QUnit.test('position should be recalculated after target option runtime change', function(assert) {
-        const $element = $('<div>');
+        const $element = $('<div>').css({
+            position: 'absolute',
+            left: 100
+        });
+        const onShowingSpy = sinon.spy();
 
         try {
             $element.appendTo('#qunit-fixture');
+            this._validationMessage.on('showing', onShowingSpy);
 
             this._validationMessage.option('target', $element);
 
-            assert.strictEqual(this._validationMessage.option('position').of, $element, 'position is recalculated');
+            assert.strictEqual(onShowingSpy.callCount, 1, 'position is recalculated');
```

Comment:
Why did u change this test? I think its initial meaning is lost

Reply:
because at present, options does not mobified by 'target'. target sets to _position field.
onShowingSpy.callCount is equal to one, because I subscribed on 'showing' event after initial show.

Reply:
Why does `showing` event raise guarantee us position is recalculated? Seems like this tests should be written another way. For example, check visual position using `getBoundingClientRect`

Reply:
rewirked